test/rest-api/request/body/body-text.node.test.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @jest-environment node
+ */
+import fetch from 'node-fetch'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import { encodeBuffer } from '@mswjs/interceptors'
+
+const server = setupServer(
+  rest.post('http://localhost/resource', async (req, res, ctx) => {
+    const body = await req.text()
```

Comment:
The intention is that we send all sorts of payload in the original request but read it as text in the handler. Then, we return the read text as the mocked response, and assert it in the test.